src/store/Store.js

```diff
@@ -123,6 +123,15 @@ class Store {
   getState() {
     return this.state;
   }
+  
+  
+  /**
+   * Stub function to stay consistent with Redux Store API. No-op.
+   * @param {object} nextReducer The reducer for the store to use instead.
+   */
+  replaceReducer(nextReducer) {
```

Comment:
Ah good call

Reply:
Seems like the linter is complaining about the arg not being used as well as some whitespace stuff:

https://travis-ci.org/tshaddix/react-chrome-redux/builds/323888944?utm_source=github_status&utm_medium=notification

Reply:
Got it. Just fixed that up.